cogs/yourlsClient/helpers.py

```diff
@@ -0,0 +1,52 @@
+import discord
+from redbot.core.utils import AsyncIter, chat_formatting
+
+
+async def createSimplePages(
+    items: [str], embedTitle: str = "", embedAuthor: discord.Member = None
```

Comment:
Pylance check: `list` expression is not allowed in type annotation.

Import `List` from `typing`, and then
```suggestion
    items: List[str], embedTitle: str = "", embedAuthor: discord.Member = None
```

Reply:
Thanks, let's also fix `cogs/tags/helper.py` afterwards as well.